src/app/features/components/periodic-edit-dialog/periodic-edit-dialog.component.html

```diff
@@ -0,0 +1,65 @@
+<h2 mat-dialog-title>Edit {{ data.selectedPeriodic.name }}</h2>
+<mat-dialog-content>
+    <form #myForm="ngForm" class="edit-periodic-form">
+        <mat-form-field>
+            <mat-label>Position</mat-label>
+            <input
+                matInput
+                [(ngModel)]="selectedPeriodic().position"
```

Comment:
That's not correct, destruct this object into single models for proper usage of Angular Signals.